closes #33 fix test for new design

danjwinter opened the pull request:
Make home page dynamic to account for any categories in database
closes #33

danjwinter committed:
fix test for new design

Make home page dynamic to account for any categories in database

danjwinter committed:
refactor category creation in test to create_list)

ToniRib commented on app/controllers/home_controller.rb:
```diff
@@ -1,5 +1,6 @@
 class HomeController < ApplicationController
   def index
+    @category_names = Category.all.pluck(:name)
```
This comment is for this line and the similar line in the test file: You don't actually need to call "all" here, `Category.pluck` will work just as well.

However, these should be actual links to the categories, so I'm going to try to make that update.

ToniRib committed:
Use jQuery to redirect to a category from the homepage

ToniRib committed:
jQuery jump to category

From either the homepage or a category show page, the links actually
work to jump to a category, including from the select boxes. This
is implemented with jQuery.

ToniRib committed:
Fix styling on category show page

ToniRib commented:
Test is still passing but now we are actually using jQuery to dynamically link to the category show pages.

ToniRib commented on app/assets/stylesheets/_custom.scss:
```diff
@@ -110,7 +115,7 @@ button.stripe-button-el span {
 .job-partial {
   border-bottom: 0.5px solid $black;
-  margin: 0 70px 40px 0;
+  margin: 0 0 40px 0;
```
I'll make this update later.

jwperry commented:
Looks solid!
